docs/reference/eql/syntax.asciidoc

```diff
@@ -382,22 +382,32 @@ use a regular string with the `\"` escape sequence.
 [[eql-syntax-wildcards]]
 === Wildcards
 
-For string comparisons using the `:` operator, you can use wildcards (`*`) to
-match specific patterns:
+For string comparisons using the `:` operator, you can use the `*` and `?`
+wildcards to match specific patterns. The `*` wildcard matches zero or more
+characters:
 
 [source,eql]
 ----
-field : "f*o"
-field : "*foo"
-field : "foo*"
+my-field : "doc*"  // Matches "doc", "docs", or "document" but not "dos"
```

Comment:
can you switch `my-field` to `my_field`? since `-` isn't a valid identifier (typically means subtraction), it would have to be `` `my-field` ``

Reply:
:doh: Good catch. Changed to `my_field`.